blog/tests/admin/test_admin_image_upload_views.py

```
@@ -0,0 +1,123 @@
+        data={"upload": content_image_file},
+    )
+
+    assert response.status_code == 200
+    data = response.json()
+    assert data["uploaded"] == 1
+    assert "url" in data
+    assert PostImage.objects.filter(post__isnull=True).count() == 1
+
+
+@pytest.mark.django_db
+def test_admin_image_upload_requires_post_method(client, staff_user):
```

Ruff (0.15.5)

[warning] Unused function argument: `staff_user`

(ARG001)